wit/deps/golem-1.x/golem-host.wit

```diff
@@ -239,6 +240,14 @@ interface host {
 
     /// Fork a worker to another worker at a given oplog index
     fork-worker: func(source-worker-id: worker-id, target-worker-id: worker-id, oplog-idx-cut-off: oplog-index) -> ();
+
+    /// Schedule invocation of any worker for later
+    schedule-invocation: func(
```

Comment:
two notes:
1: idempotency keys will be generated by the executor rn. Do we need to allow users to provide their own?
2: wit-value is a bit annoying to construct. Any better ideas what to use instead?